dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/IndiciesFactory.java

```diff
@@ -80,9 +81,10 @@ public void point(final IndiciesInfo newInfo) throws DotDataException {
         DotConnect dc = new DotConnect();
         final String insertSQL = "INSERT INTO indicies VALUES(?,?)";
         final String deleteSQL = "DELETE from indicies where index_type=? or index_name=?";
-        for (IndexTypes type : IndexTypes.values()) {
+        for (IndexType type : IndexType.values()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'type' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)